social-network/src/missedArticle.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+/*import './article.css';*/
+import './flexstyle.css';
+import PropTypes from 'prop-types';
+
+class Missed_article extends Component {
+    render(){
+     /*   const{ //shove all data from json pull here.
+            title,
+            description,
+            image,
+            author, //Author object (Want name & image)
+            link
+        } = this.props.article; */
+        console.log(this.props.article); //for debugging
+        return (
+            <div className='Missed_article-container'> 
+                <div className="Article-img2">
+                    <img className = 'theImage2' alt='missedCover' src={this.props.article.image}/> 
+                </div>   
+                <div className='nonImageStuff2'>   
+                    <p className='Article-title' >{this.props.article.title}</p>
+                    <h4 className='Article-descripton2'>{this.props.article.description}</h4>
+                    <div className='Author2'>
+                        <img className='authorImage' alt='missedCover' src={this.props.article.author.image}/>
+                        <p className='authorName'>{this.props.article.author.name}</p>
+                        
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
+
+Missed_article.propTypes = {
```

Comment:
Make sure to use camelcase when naming components. Ex. `MissedArticle`. 

Additionally, the proptypes for both article components are a little off. The components use `image`, `title`, or `description`, but they're all based on `article`. So you'll want to refactor your proptypes so that these props are nested under article. Ex.

```js
MissedArticle.propTypes = {
  article: PropTypes.shape({
    title: PropTypes.string,
    ...
  })
};
```